src/utils/get-command-spec.test.ts

```diff
@@ -19,10 +19,16 @@ describe('#getCommandSpec()', () => {
 
 	it('Complains about invalid spec JS', async () => {
 		await expect(getCommandSpec(path.join(testFileTrees, 'bad-structure', 'cli', 'invalid-spec-js'))).rejects.toThrow(
-			'Encountered this error while importing the spec file at'
+			'ReferenceError: bad is not defined'
```

Comment:
Why is this being changed?

Reply:
The content I moved it towards is more specific than the old text.  There could be other reasons for failure and still appear to pass.

Reply:
This seems like something that should be addressed in a separate issue/PR. Can you open an issue for this separately?

Reply:
Ok.  I opened: https://github.com/sparksuite/waterfall-cli/issues/378